src/content/whats-new/2022/03/whats-new-03-01-Kubernetes-APM.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Minimize Kubernetes complexity to build performant applications'
+summary: 'Analyze application and Kubernetes observability data in one place'
+releaseDate: '2022-03-01'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/kubernetes-experience-in-APM'
+---
+## See Kubernetes details in APM
+
+Use our new UI that shows how cluster performance impacts your applications.
```

Comment:
I'd finish with :

Reply:
@urbiz-nr: I'm not sure you finished your comment. How would you finish this sentence?

Reply:
I mean I'd finish the sentence with : - as in Use our new UI that shows how cluster performance impacts your applications:

Reply:
Not super important in any case :)

Reply:
@rhetoric101